client/tokenized-express-checkout/transformers/wc-to-stripe.js

```diff
@@ -96,7 +96,7 @@ export const transformCartDataForDisplayItems = ( cartData ) => {
  * @return {{id: string, label: string, amount: integer, deliveryEstimate: string}} `shippingRates` for Stripe.
  */
 export const transformCartDataForShippingRates = ( cartData ) =>
-	cartData.shipping_rates?.[ 0 ].shipping_rates
```

Comment:
I noticed that sometimes `shipping_rates` was an empty array (and not just `undefined`). So, fixing.